### DISCUSSION TITLE
Use GTK frontends for mpv

### DISCUSSION THREAD

**Discussion by ghost** [pystardust/ani-cli#1001] [Q&A] [answered]
I'm not a big fan of the default look of mpv when watching stuff, can I use a frontend?
It would look a LOT nicer using Clapper or Celluloid.

I understand this may be not possible but I'm just curious about it, this is my favorite way of watching anime and I believe the devs did a great job!

**Comment by 71zenith** [comment, marked as answer]
yes, u can.
`export ANI_CLI_PLAYER=celluloid` put this in ur `.bashrc`, `.zshrc` or any other startup script that ur shell launches
otherwise u can run it directly like this `ANI_CLI_PLAYER=celluloid ani-cli`

**Reply by ghost** [reply]
It's just copy and paste or I have to do something specific inside .bashrc and .zshrc?

**Reply by 71zenith** [reply]
just copy paste `export ANI_CLI_PLAYER=celluloid`
